src/lib/AnnotationControls.ts

```diff
@@ -153,7 +153,7 @@ export default class AnnotationControls {
         const regionButton = this.controls.add(
             __('region_comment'),
             this.handleClick(onRegionClick, AnnotationMode.REGION),
-            `${CLASS_BOX_CONTROLS_GROUP_BUTTON} ${CLASS_REGION_BUTTON}`,
+            `${CONTROLS_BUTTON_CLASS} ${CLASS_REGION_BUTTON}`,
```

Comment:
`CONTROLS_BUTTON_CLASS` is added in `Controls.js`, right? I think in the future, there will be more annotations buttons. Do we want to remove the group class?

Reply:
I don't think the group class is what we need with the individual buttons.

Reply:
Is `CONTROLS_BUTTON_CLASS` needed? I believe `this.controls` already has it.